cmd/kpod/load.go

```diff
@@ -90,20 +82,7 @@ func loadCmd(c *cli.Context) error {
 		}
 	}
 
-	opts := loadOptions{
-		input: input,
-		quiet: quiet,
```

Comment:
Since this is not needed anymore, the struct definition at the beginning can be removed as well.

Reply:
Removed